.circleci/config.yml

```diff
@@ -883,11 +963,11 @@ jobs:
       - store_test_results:
           path: test/test-results/e2e.xml
 
-  test-e2e-chrome-snaps:
+  test-e2e-chrome-snaps-flask:
     executor: node-browsers
     parallelism: 4
     steps:
-      - run: *shallow-git-clone
+      - checkout
```

Comment:
Could we use a shallow clone here as well?

Reply:
For sure! I must have missed that when rebasing!